test-e2e/test_basic_switch.py

```diff
@@ -16,12 +16,18 @@ def setUp(self):
 
     def tearDown(self):
         logger.info("Tearing down the test")
-        self.api.delete_pod("ep6")
-        self.api.delete_pod("ep7")
+        #self.api.delete_pod("ep6")
+        #self.api.delete_pod("ep7")
 
     def test_basic_switch(self):
         logger.info("Testing testing testing :)")
         if self.ep1.ping(self.ep2.ip):
-            logger.info("Success")
+            logger.info("Ping test from {} to {} Success".format(self.ep1.name, self.ep2.name))
         if self.ep2.ping(self.ep1.ip):
-            logger.info("Success")
+            logger.info("Ping test from {} to {} Success".format(self.ep2.name, self.ep1.name))
+        if self.ep1.http(self.ep2.name):
+            logger.info("Http test from {} to {} Success".format(self.ep1.name, self.ep2.name))
+        if self.ep2.http(self.ep1.name):
+            logger.info("Http test from {} to {} Success".format(self.ep2.name, self.ep1.name))
+        if self.ep1.tcp(self.ep2.name):
+            logger.info("TCP test from {} to {} Success".format(self.ep1.name, self.ep2.name))
```

Comment:
We will be doing these similar test times and again. Please consolidate in one method like ```do_common_tests``` here https://github.com/futurewei-cloud/mizar/blob/dev-next/test/trn_func_tests/helper.py#L98-L102